src/Microsoft.AspNet.Authentication.Facebook/FacebookHandler.cs

```diff
@@ -38,7 +38,7 @@ public FacebookHandler(HttpClient httpClient)
 
             var payload = JObject.Parse(await response.Content.ReadAsStringAsync());
 
-            var context = new OAuthCreatingTicketContext(new ClaimsPrincipal(identity), properties, Context, Options, Backchannel, tokens, payload);
+            var context = new OAuthCreatingTicketContext(new AuthenticationTicket(new ClaimsPrincipal(identity), properties, Options.AuthenticationScheme), Context, Options, Backchannel, tokens, payload);
```

Comment:
This line is so looooooooong :smile: 

(you could creating a `ticket` variable to reduce its length: https://github.com/aspnet-contrib/AspNet.Security.OpenId.Providers/blob/dev/src/AspNet.Security.OpenId/OpenIdAuthenticationHandler.cs#L216-L230)